src/prefect/runner/server.py

```diff
@@ -0,0 +1,39 @@
+import pendulum
+import uvicorn
+from fastapi import APIRouter, FastAPI, status
+from fastapi.responses import JSONResponse
+from functools import partial
+
+
+def perform_health_check(runner, delay_threshold: int = 15) -> JSONResponse:
```

Comment:
Since a runner's poll frequency is configurable, having the `delay_threshold` as a multiple of the runner's poll frequency would be cool.

Reply:
Yea that's smart, maybe twice the poll frequency would be the most robust